views/update/relevant-period-interrupt-card.html

```diff
@@ -0,0 +1,47 @@
+{% extends "update-layout.html" %}
+{% import "includes/date-macros.html" as dateMacros %}
+{% set start_date = "28 February 2022" %}
+{% set registration_date = dateMacros.formatDate(dateOfCreation["day"], dateOfCreation["month"], dateOfCreation["year"]) %}
```

Comment:
Shouldn't need these dateMacros, start date, or registration date. It's all handled inside `relevant-period.html` now

Reply:
changed as suggested